### ISSUE TITLE
How to install these?

### ISSUE THREAD

**Issue by DerRolo** [xan1242/NFS-XtendedInput#79]
Hello, I think I'm stupid, but I do not understand what should I do with these mass of data.

I want to play NFS Carbon (PC Version) completely with controller and have the right Buttons layouts there.

Should the NFS_XtendedInput.default.ini, NFS_XtendedInput.ini & EventReference.txt in the scripts folder? It that all? But this is for me not working, nothing happend.

Thank you for your time!

**Comment by Lumanator** [comment]
Did you make sure to have the Ultimate ASI Loader be in the root folder?

**Comment by xan1242** [comment]
Yeah, sorry for not providing an installation tutorial (I'm working on some other things surrounding this project).

Basically, the release packages should come with everything you need.

All you need to do is extract all of the files and folders from the zip to the game installation directory (right next to the game exe file). It should work immediately.

So, dinput8.dll and scripts folder should sit next to NFSC.exe for example.

In case it doesn't work after that, it might be due to some file permission issues (e.g. the game is installed in Program Files). In this case, you could move the game out of there to another location (like the desktop or documents folder, or another drive entirely) and try again.

Also, make sure you're using version 1.4 with a US 1.4 noce/crack exe as it will only work with that. (This is planned to be partly fixed with a pattern detector later)

**Comment by lulutov** [comment]
I'm really confused. I must not have installed it correctly, because B is still A and X is still B and so on. I extracted only the two .ini files from the Carbon mod folder and placed them in the game folder next to the game's .exe.

> All you need to do is extract all of the files and folders from the zip to the game installation directory (right next to the game exe file). It should work immediately.

ALL of the files? or is it just the two .ini files from the corresponding game folder?

> In case it doesn't work after that, it might be due to some file permission issues (e.g. the game is installed in Program Files). In this case, you could move the game out of there to another location (like the desktop or documents folder, or another drive entirely) and try again.

I did this, no dice.

I reset the controls to default, ran the game as admin, etc

**Comment by Naiyan27** [comment]
> I'm really confused. I must not have installed it correctly, because B is still A and X is still B and so on. I extracted only the two .ini files from the Carbon mod folder and placed them in the game folder next to the game's .exe.
> 
> > All you need to do is extract all of the files and folders from the zip to the game installation directory (right next to the game exe file). It should work immediately.
> 
> ALL of the files? or is it just the two .ini files from the corresponding game folder?
> 
> > In case it doesn't work after that, it might be due to some file permission issues (e.g. the game is installed in Program Files). In this case, you could move the game out of there to another location (like the desktop or documents folder, or another drive entirely) and try again.
> 
> I did this, no dice.
> 
> I reset the controls to default, ran the game as admin, etc

Same here. Can't get it to work at all. So confused on what to do.
